tox.ini

```diff
@@ -73,19 +73,17 @@ deps =
 
     # The oldestdeps factor is intended to be used to install the oldest versions of all
     # dependencies that have a minimum version.
-    oldestdeps: pyerfa==2.0.*
+    oldestdeps: pyerfa==2.0.1.1
     oldestdeps: numpy==1.23.*
-    oldestdeps: matplotlib==3.3.*
+    oldestdeps: matplotlib==3.6.*
```

Comment:
I just checked the logs and saw that our oldest dependencies tests have indeed been using the latest `matplotlib`, but this change fixes it.